app/jobs/base.rb

```diff
@@ -461,6 +470,50 @@ def self.enqueue_at(datetime, job_name, opts = {})
     enqueue_in(secs, job_name, opts)
   end
 
+  def self.decrease_batch_count(batch_id)
```

Comment:
Let's mark this as private or at least `:nodoc:` so that it is obvious this isn't a public API.